components/organisms/Footer.js

```diff
@@ -0,0 +1,85 @@
+import PropTypes from "prop-types";
+
+/**
+ * footer element for all pages
+ */
+
+export function Footer(props) {
+  return (
+    <div className="w-full mb-4">
+      <div className="w-full h-auto  bg-dk-blue bg-footer-parliament-image bg-no-repeat bg-right-bottom ">
+        <div className="w-full  py-6 layout-container  ">
+          <ul className="flex flex-wrap justify-between ">
+            {" "}
+            {props.footerBoxLinks.map((value, index) => {
+              return (
+                <li
+                  key={index}
+                  className="text-white w-64 md:w-56 lg:w-80   my-2   hover:underline"
+                >
+                  <a href={value.footerBoxlink}>{value.footerBoxLinkText}</a>
+                </li>
```

Comment:
These links look really good, almost there.

3 things I would point out are 

- our link text should be smaller on smaller screens (in the screenshot you can see the Canada.ca footer has smaller link text for the white links).
- there is no 3-column layout for Canada.ca design: it goes from 2 columns to 1. I think this is resolved by making the `li`s wider.
- Our links get a lot closer to the right side of the screen (even touching it on smaller screen sizes), so we should make sure they don't get that far.

Here is a side-by-side screenshot. Our site is on the left, Canada.ca is on the right.

<img width="1437" alt="Screen Shot 2021-04-09 at 11 07 02" src="https://user-images.githubusercontent.com/2454380/114201201-f21faf80-9923-11eb-9717-d0911c54c52c.png">